sirepo/job_api.py

```diff
@@ -69,6 +71,14 @@ def api_downloadDataFile(simulation_type, simulation_id, model, frame, suffix=No
         )
 
 
+@api_perm.require_user
+def api_jobs():
```

Comment:
This should be prefixed with something. There should be admJobs and maybe myJobs or ownJobs. The auth check should be first entry point for admJobs. 

Partitioning name spaces is important, because we may want to restrict access to particular APIs by IP in the proxy. We do this already [in rsconf for job supervisor](https://github.com/radiasoft/rsconf/blob/03b5d9d8cd905a917872721239bfd3461571436b/rsconf/package_data/sirepo_job_supervisor/nginx.conf.jinja#L7).